src/pages/storage/actions/DeleteStorageVolumeBtn.tsx

```diff
@@ -103,7 +105,12 @@ const DeleteStorageVolumeBtn: FC<Props> = ({
       className={classname}
       shiftClickEnabled
       showShiftClickHint
-      disabled={Boolean(disabledReason)}
+      disabled={!canDeleteVolume(volume) || Boolean(disabledReason)}
+      title={
```

Comment:
```suggestion
      onHoverText={
```
The title prop gets ignored and we should use on hover text instead to override it.